helpers/helper-only-numbers/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@brazilian-utils/helper-only-numbers",
+  "version": "1.0.0-alpha.1",
+  "author": "Hyan Mandian <hyanmandian@hotmail.com>",
+  "repository": "https://github.com/brazilian-utils/brazilian-utils/tree/master/packages/helper-only-numbers",
+  "bugs": {
+    "url": "https://github.com/brazilian-utils/brazilian-utils/issues"
+  },
+  "license": "MIT",
+  "main": "dist/index.js",
```

Comment:
maybe missing umd:main?

Reply:
Is it worthy? UMD bundle are bigger and I think using bundlers like react or rollup are pretty standard today.

Look at this issue: https://github.com/palmerhq/tsdx/issues/120

Reply:
I agree with you, so maybe we should remove all umb bundle for our packages. What do you think about it?

Reply:
Yep. This PR doesn't include a UMD bundle.